cclient/serialization/tests/test_json.c

```diff
@@ -333,14 +366,20 @@ void test_deserialize_get_trytes(void) {
       "{\"trytes\":["
       "\"" GET_TRYTES_DESERIALIZE_TX "\"]}";
 
-  get_trytes_res_t* tips = get_trytes_res_new();
+  trit_array_p tmp_trytes = NULL;
+  trit_array_p hash1 =
+      trit_array_new_from_trytes((tryte_t*)GET_TRYTES_DESERIALIZE_TX);
```

Comment:
Can we change the name of the hashes/transaction's trytes to not be command specific?
(e.g TX_1_HASH_TRYTES)

Reply:
will do it in a refactor PR.